v2/internal/config/vars_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"reflect"
 	"testing"
 
+	// TODO: Importing this for side effects is required as it initializes cloud.AzurePublic
```

Comment:
Don't think this needs `TODO`.
```suggestion
	// Importing this for side effects is required as it initializes cloud.AzurePublic
```

Reply:
Good point